src/agent/onefuzz/src/libfuzzer.rs

```diff
@@ -211,7 +211,7 @@ impl<'a> LibFuzzer<'a> {
         let output_text = String::from_utf8_lossy(&output.stderr);
         let pat = r"MERGE-OUTER: (\d+) new files with (\d+) new features added";
         let re = regex::Regex::new(pat).unwrap();
-        match re.captures_iter(&output_text).next() {
+        let result = match re.captures_iter(&output_text).next() {
```

Comment:
It might address the compiler complaint a bit more directly if we were to do:
```suggestion
        let captures = re.captures_iter(&output_text).next();
        match captures {
```